docs/restrict-cross-namespace-associations.asciidoc

```diff
@@ -0,0 +1,97 @@
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/restrict-cross-namespace-associations.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-restrict-cross-namespace-associations"]
+=== Restrict cross namespace resource associations
+
+By default, associations can be created between any resources as long as they are deployed in some namespaces managed by the same ECK instance.
+
+This is true, for example, when using the `elasticsearchRef` field in order to automatically configure and establish a trust relationship between Kibana and Elasticsearch. If a user can create a Kibana instance in a namespace managed by ECK then this Kibana instance can be associated with any Elasticsearch cluster deployed in another namespace managed by the same ECK instance.
```

Comment:
```suggestion
This is true, for example, when using the `elasticsearchRef` field in order to automatically configure and establish a trust relationship between Kibana and Elasticsearch. If a user can create a Kibana instance in a namespace managed by ECK, then this Kibana instance can be associated with any Elasticsearch cluster deployed in another namespace managed by the same ECK instance.
```

Reply:
It might also be worth calling out that this is possible by default even if the user only has permissions to edit the Kibana and not the Elasticsearch resource

Reply:
- in order to... => to
- If a user can => if you can

Reply:
@alaudazzi  re. "user" vs "you" and the use of the passive voice:
I think this part is dedicated to someone who is administering ECK. In this particular situation I wonder if it is not better to keep the term "user" because the idea here is to explain to the admin: "This is what your users can do by default"

Happy to change my mind though :)